evaluation/cron/cloud_build.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'eu.gcr.io/robco-klose/eval_nightly', '.']
```

Comment:
should your gCloud account be visible here? can we use `robco-cartographer`?

Reply:
not yet. I'll move everything to the cartographer cloud project soon and will write up a little more in the README.md about the configuration steps required while doing so.

Reply:
ACK